xcodebuild/build.go

```diff
@@ -30,8 +29,18 @@ xcodebuild -workspace <workspacename> \
 	[<buildaction>]...
 */
 
-// ArchiveCommandModel ...
-type ArchiveCommandModel struct {
+// const ...
+const (
+	Archive BuildCommand = "archive"
+	Build   BuildCommand = "build"
+	Analyze BuildCommand = "analyze"
+)
+
+// BuildCommand can be {archive, build, analyze}
+type BuildCommand string
+
+// BuildCommandModel ...
+type BuildCommandModel struct {
```

Comment:
simply Command? it would look nice when it is called: `xcodebuild.Command`

Reply:
I agree but it will conflict with the xcodebuild.go/CommandModel interface{}.
I'm thinking about it.